src/ipc/get-usb-drives.ts

```diff
@@ -51,6 +62,19 @@ async function getUsbDrives(): Promise<UsbDrive[]> {
         }
       }),
     )
+    const currentMountPoints = usbDrives.map(drive => drive.mountPoint)
+
+    // Find any phantom usb drives that were inproperly removed and need to be unmounted
+    const { stdout } = await exec('findmnt', ['-J', '-t', 'vfat'])
```

Comment:
I filter for the type vfat as otherwise I got a lot of random unrelated things, they will fail the regexp check anyway but I thought this felt safer... I'm not sure if there are other types we should allow in addition to vfat though.

Reply:
that's a good default, though I can imagine supporting other drive formats in the future. Let's see how this works for now.

Reply:
If we're formatting the drives ourselves and are testing them to make sure this works, I think it's probably okay. That said, I don't think we need to filter:

```js
const { execFileSync } = require('child_process')
const { existsSync } = require('fs')

const stdout = execFileSync('findmnt', ['-J', '--list'])
const { filesystems } = JSON.parse(stdout)

for (const { source, target } of filesystems) {
  if (!source.startsWith('/')) {
    continue
  }

  if (!existsSync(source)) {
    console.log(`${source} → ${target}`)
  }
}
```

On my machine this prints nothing when no drives have been improperly ejected. If I _do_ improperly eject something I get an entry like this:

```
/dev/sdb1 → /media/usb-drive
```

So I think we can just filter the list using something like this to unmount the phantom ones, though I think we should make it a separate call.